MapboxCoreNavigation/CLLocation.swift

```diff
@@ -30,10 +30,10 @@ extension CLLocation {
     convenience init(_ location: MBFixLocation) {
         self.init(coordinate: location.coordinate,
                   altitude: location.altitude?.doubleValue ?? 0,
-                  horizontalAccuracy: location.accuracyHorizontal?.doubleValue ?? 0,
+                  horizontalAccuracy: location.accuracyHorizontal?.doubleValue ?? -1,
                   verticalAccuracy: 0,
-                  course: location.bearing?.doubleValue ?? 0,
-                  speed: location.speed?.doubleValue ?? 0,
+                  course: location.bearing?.doubleValue ?? -1,
+                  speed: location.speed?.doubleValue ?? -1,
```

Comment:
when they are missing from `MBFixLocation` we mark as invalid in the `CLLocation`